pkg/repository/testworkflow/mongo.go

```diff
@@ -255,16 +260,36 @@ func (r *MongoRepository) GetExecutionsSummary(ctx context.Context, filter Filte
 		"result.steps":          0,
 		"result.initialization": 0,
 		"workflow.spec":         0,
-		"resolvedWorkflow":      0,
 	})
 	cursor, err := r.Coll.Find(ctx, query, opts)
 	if err != nil {
 		return
 	}
-	err = cursor.All(ctx, &result)
+	err = cursor.All(ctx, &executions)
+	result = make([]testkube.TestWorkflowExecutionSummary, len(executions))
+	for i := range executions {
+		executions[i].UnscapeDots()
 
-	for i := range result {
-		result[i].UnscapeDots()
+		if executions[i].ResolvedWorkflow != nil && executions[i].ResolvedWorkflow.Spec != nil {
+			for k, v := range executions[i].ResolvedWorkflow.Spec.Config {
+				if v.Default_ != nil {
+					if executions[i].ConfigParams == nil {
+						executions[i].ConfigParams = make(map[string]testkube.TestWorkflowExecutionConfigValue)
+					}
```

Comment:
It should ignore then, as in such case it means that the passed configuration was ignored (i.e. sensitive)

Reply:
No ConfigParams should not be `nil`, but rather `map[string]string{}`.